src/ts/components/Controls.tsx

```diff
@@ -58,6 +61,13 @@ class Controls extends Component<IProps, IState> {
             >
               Check document
             </button>
+          
+            <IconButton
+              aria-label="close Typerighter"
+              onClick={this.props.deactivate}
```

Comment:
I think this icon would look a little better if it was set to small using the size prop: https://material-ui.com/api/icon-button/

Reply:
> The code changes look good, just a couple of comments on the UI. I would also favour Typerighter being active by default but I would be interested to hear your thoughts on that.

Yep, this is wired to start as 'off' for testing - it will be eventually set according to the isActive param in state when https://github.com/guardian/prosemirror-typerighter/pull/91/ is merged.